modules/juce_audio_devices/native/juce_ios_Audio.cpp

```diff
@@ -965,7 +965,9 @@ struct iOSAudioIODevice::Pimpl      : public AudioPlayHead,
         desc.componentManufacturer = kAudioUnitManufacturer_Apple;
         desc.componentFlags = 0;
         desc.componentFlagsMask = 0;
-
+                
+        setAudioPreprocessingEnabled(isUsingBuiltInSpeaker());

```

Comment:
In theory we might also be able to do this without changing the juce fork since this is a public method, but I'm ok with doing it here for now.